README.md

```diff
@@ -40,3 +40,75 @@ if ($file->getSize()) {
 // Access a resource provided by this Divide.IQ server.
 $result = $divideIq->request('stockbase_stock');
 ```
+
+Example Usage orderRequest POST
+===============================
+  
+```php
+use DivideBV\PHPDivideIQ\DivideIQ;
+use DivideBV\PHPDivideIQ\Models;
+ 
+
+$person = new Models\Person(); // Requires surname of customer.
+$address = new Models\Address(); // Requires Street, StreetNumber, Zipcode, City and CountryCode in ISO 3166-1 alpha-3 format. (e.g.: NLD)
+$orderDelivery = new Models\OrderDelivery(); // Holds Person and Address to deliver.
+$orderLine1 = new Models\Orderline(); // Holds EAN, Amount, and Number of the number of orderline (1).
+$orderLine2 = new Models\Orderline(); // Holds EAN, Amount, and Number of the number of orderline (2).
+$orderHeader = new Models\OrderHeader(); // Holds orderNumber, Timestamp, and additional information.
+$orderRequest = new Models\OrderRequest(); // Holds orderHeader, OrderLines, and OrderDelivery
+ 
+$person->setGender('Male');
+$person->setInitials('J.');
+$person->setFirstName('John');
+$person->setSurname('Doe');
+$person->setCompany('JohnDoeCompany');
+ 
+$address->setStreet('Examplestreet');
+$address->setStreetNumber('33');
+$address->setStreetNumberAdditition('b');
+$address->setZipcode('1234AB');
+$address->setCity('Amsterdam');
+$address->setCountryCode('NLD');
+ 
+$orderLine1->setNumber(1); // Line number
+$orderLine1->setEan('2000000000003'); // EAN
+$orderLine1->setAmount(3); // Amount/Quantity
+ 
+$orderLine2->setNumber(2);
+$orderLine2->setEan('2000000000002');
+$orderLine2->setAmount(10);
+ 
+$orderHeader->setOrderNumber(40001);
+$orderHeader->setAttention("Testorder");
+$orderHeader->setTimestamp(new DateTime());
+ 
+$orderDelivery->setPerson($person);
+$orderDelivery->setAddress($address);
+ 
+// OrderLines are not implemented to support REST yet
+// $orderRequest->addOrderLine($orderLine1);
+// $orderRequest->addOrderLine($orderLine2);
+ 
+$orderRequest->setOrderHeader($orderHeader);
+$orderRequest->setOrderDelivery($orderDelivery);
+
+ 
+header('Content-Type: application/json');
+ 
+// Try to do the POST with orderRequest Model.
+try {
+    $postResponse = $divideIq->request('stockbase_orderrequest', $orderRequest->toArray(), 'POST');
```

Comment:
Implemented JsonSerializable and adjusted the Readme in commit: 
https://github.com/DivideBV/PHPDivideIQ/pull/1/commits/19da445b970041225a00f424ad3f2597142fb493

https://github.com/Nav-Appaiya/PHPDivideIQ/commit/e684ca9f1e8a3a39f67f7236e78b1174ac5dfbc8